src/main/java/net/morbz/osmonaut/osm/LatLon.java

```diff
@@ -28,59 +28,62 @@
 
 /**
  * A class that represents the latitude and longitude.
+ * 
  * @author MorbZ
  */
 public class LatLon {
-	private double lat;
-	private double lon;
+    private double lat;
+    private double lon;
 
-	/**
-	 * @param lat The latitude
-	 * @param lon The longitude
-	 */
-	public LatLon(double lat, double lon) {
-		this.lat = lat;
-		this.lon = lon;
-	}
-	
-	/**
-	 * @return The latitude
-	 */
-	public double getLat() {
-		return lat;
-	}
+    /**
+     * @param lat
+     *            The latitude
+     * @param lon
+     *            The longitude
+     */
+    public LatLon(double lat, double lon) {
+        this.lat = lat;
+        this.lon = lon;
+    }
 
-	/**
-	 * @return The longitude
-	 */
-	public double getLon() {
-		return lon;
-	}
-	
-	/**
-	 * {@inheritDoc}
-	 */
-	@Override
-	public boolean equals(Object obj) {
-		if(!(obj instanceof LatLon)) {
-			return false;
-		}
-		LatLon latlon = (LatLon)obj;
-		if(latlon.getLat() != lat) {
-			return false;
-		}
-		if(latlon.getLon() != lon) {
-			return false;
-		}
-		return true;
-	}
-	
-	/**
-	 * {@inheritDoc}
-	 */
-	@Override
-	public String toString() {
-		return "{ lat: " + StringUtil.formatCoordinate(lat) + 
-				", lon: " + StringUtil.formatCoordinate(lon) + " }";
-	}
+    /**
+     * @return The latitude
+     */
+    public double getLat() {
+        return lat;
+    }
+
+    /**
+     * @return The longitude
+     */
+    public double getLon() {
+        return lon;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean equals(Object obj) {
+        if (!(obj instanceof LatLon)) {
+            return false;
+        }
+        LatLon latlon = (LatLon) obj;
+        if (latlon.getLat() != lat) {
+            return false;
+        }
+        if (latlon.getLon() != lon) {
+            return false;
+        }
+        return true;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String toString() {
+        return "{ lat: " + StringUtil.formatCoordinate(lat) +
+                ", lon: " + StringUtil.formatCoordinate(lon) + " }";
+    }
 }
```

Comment:
@MorbZ Do you happen to have by chance an auto-formatter configuration we can use to ensure consistent formatting? (Eclipse for example).